vmdb/app/models/miq_provision_configured_system_foreman_workflow.rb

```diff
@@ -0,0 +1,41 @@
+class MiqProvisionConfiguredSystemForemanWorkflow < MiqProvisionConfiguredSystemWorkflow
+  def self.default_dialog_file
+    'miq_provision_configured_system_foreman_dialogs'
+  end
+
+  def create_request(values, requester_id, auto_approve = false)
+    event_message = "Provision requested by [#{requester_id}] for Configured Systems:#{values[:src_configured_system_ids].inspect}"
+    super(values, requester_id, 'ConfiguredSystem', 'configured_system_provision_request_created', event_message, auto_approve)
+  end
+
+  def update_request(request, values, requester_id)
+    event_message = "Provision request successfully updated by [#{requester_id}] for Configured Systems:#{values[:src_configured_system_ids].inspect}"
+    super(request, values, requester_id, 'ConfiguredSystem', 'configured_system_provision_request_updated', event_message)
+  end
+
+  def get_source_and_targets(_refresh = false)
+  end
+
+  def update_field_visibility
+  end
+
+  # Methods for populating lists of allowed values for a field
+
+  def allowed_configured_systems(_options = {})
+    @allowed_configured_systems ||= begin
+      ConfiguredSystem.all.collect do |cs|
+        build_ci_hash_struct(cs, [:hostname, :id])
+      end
+    end
+  end
+
+  def allowed_configuration_profiles(_options = {})
+    @allowed_configuration_profiles ||= begin
+      configured_system_id  = get_value(@values[:src_configured_system_ids]).first
+      configuration_manager = ConfiguredSystem.where(:id => configured_system_id).first.configuration_manager
+      configuration_manager.configuration_profiles.all.collect do |cp|
+        build_ci_hash_struct(cp, [:name, :id])
```

Comment:
Same here, you do not need to pass `id`.  What other values would be useful to display in the list-view.